.github/workflows/check-components-changelog.yml

```
@@ -0,0 +1,25 @@
+name: Verify @wordpress/components CHANGELOG update
+
+on:
+    pull_request:
+        types: [opened, synchronize]
+        paths:
+            - 'packages/components/**'
+            - '!packages/components/**/stories/**'
```

Out of curiosity, any reasons why `src` is not included in the stories ignore path?

```suggestion
            - '!packages/components/src/**/stories/**'
```

---

Oversight on my part, the more specific path makes more sense, thanks!